src/main/java/com/google/collegeplanner/data/Meeting.java

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.package com.google.collegeplanner.data;
+
+package com.google.collegeplanner.data;
+
+import java.time.DayOfWeek;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class Meeting {
+  private ArrayList<DayOfWeek> days;
+  private String room;
+  private String building;
+  private int startTime;
+  private int endTime;
+
+  public Meeting(String days, String room, String building, int startTime, int endTime) {
+    this.days = new ArrayList<String>();
+    this.room = room;
+    this.building = building;
+    this.startTime = startTime;
+    this.endTime = endTime;
+
+    if (days.toUpperCase().contains("M")) {
+      this.days.add(DayOfWeek.MONDAY);
+    }
+    if (days.toUpperCase().contains("TU")) {
+      this.days.add(DayOfWeek.TUESDAY);
+    }
+    if (days.toUpperCase().contains("W")) {
+      this.days.add(DayOfWeek.WEDNESDAY);
+    }
+    if (days.toUpperCase().contains("TH")) {
+      this.days.add(DayOfWeek.THURSDAY);
+    }
+    if (days.toUpperCase().contains("F")) {
+      this.days.add(DayOfWeek.FRIDAY);
+    }
```

Comment:
Since it's an academic calendar, I didn't really take into account the weekends as classes only take place Monday-Friday. Also, the calendar on the website is a 5-day calendar so even if there were weekend classes we would need to alter the calendar UI.

Reply:
The UI code that savion wrote handles Saturday/Sunday in the code. It's important to handle these edge cases somehow. One way to do that is to just throw an exception if you detect an "S"?